tasklist/client/e2e/pages/TaskPanelPage.ts

```diff
@@ -49,12 +49,8 @@ class TaskPanelPage {
     }
   }
 
-  task(name: string) {
-    return this.availableTasks.getByLabel(name).nth(0);
-  }
-
   async openTask(name: string) {
-    await this.task(name).click();
+    await this.availableTasks.getByText(name, {exact: true}).nth(0).click();
```

Comment:
@douglasbouttell-camunda why was this necessary?

Reply:
Trying to fix/debug the end to end tests

Reply:
E2E tests are flaky in the CI right now
I plan on checking them, but if you have spare time please go ahead

It's better to download the report from the CI to analyse them